src/api/fetchAPI.ts

```diff
@@ -0,0 +1,7 @@
+export interface fetchApi {
+    url: string;
+
+    getWorkflows: (args?:any) => Promise<any>;
+    getWorkflow: (args?:any) => Promise<any>;
```

Comment:
Can we add the parameters here? e.g.
getWorkflow: (name: string) => Promise<any>;

Reply:
Okay! My goal was to leave it as open as possible for the different future implementation. But I'm changing it to be more specific :)